internal/yggdrasil/yggdrasil.go

```diff
@@ -233,10 +241,20 @@ func getDeviceMetricsConfiguration(edgeDevice *v1alpha1.EdgeDevice) *models.Metr
 			metricsConfig.System = &models.SystemMetricsConfiguration{
 				Interval: systemMetrics.Interval,
 			}
+
+			allowListSpec := systemMetrics.AllowList
+			if allowListSpec != nil {
+				allowList, err := h.allowLists.GenerateFromConfigMap(ctx, allowListSpec.Name, allowListSpec.Namespace)
```

Comment:
Should we have some default metrics we would like to collect? We could add a flag to explicitly disable metrics collection.

Reply:
I want to have defaults on the device side.

I thought about allowing users to disable system metrics collection and I will implement it as a separate couple of PRs.